x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/fleet/setup_page/index.tsx

```diff
@@ -38,84 +67,154 @@ export const SetupPage: React.FunctionComponent<{
+                defaultMessage="In order to use Fleet, you must create an Elastic user. This user can create API keys
+        and write to logs-* and metrics-*."
+              />
+            </EuiText>
+            <EuiSpacer size="l" />
+            <EuiForm>
+              <form onSubmit={onSubmit}>
+                <EuiButton fill isLoading={isFormLoading} type="submit">
+                  <FormattedMessage
+                    id="xpack.ingestManager.setupPage.enableFleet"
+                    defaultMessage="Create user and enable Fleet"
+                  />
+                </EuiButton>
+              </form>
+            </EuiForm>
+            <EuiSpacer size="m" />
+          </EuiPageContent>
+        </EuiPageBody>
+      </WithoutHeaderLayout>
+    );
+  }
+
+  return (
+    <WithoutHeaderLayout>
+      <EuiPageBody restrictWidth={820}>
+        <EuiPageContent>
+          <EuiCallOut
+            title={i18n.translate('xpack.ingestManager.setupPage.missingRequirementsCalloutTitle', {
+              defaultMessage: 'Missing security requirements',
+            })}
+            color="warning"
+            iconType="alert"
+          >
             <FormattedMessage
-              id="xpack.ingestManager.setupPage.missingRequirementsTitle"
-              defaultMessage="Missing requirements"
+              id="xpack.ingestManager.setupPage.missingRequirementsCalloutDescription"
+              defaultMessage="In order to use Fleet, you must enable the following Elasticsearch and Kibana security features."
             />
-          </h2>
-        </EuiTitle>
-        <EuiSpacer size="xl" />
-        <EuiText color="subdued" textAlign={'left'}>
+          </EuiCallOut>
+          <EuiSpacer size="m" />
           <FormattedMessage
-            id="xpack.ingestManager.setupPage.missingRequirementsDescription"
-            defaultMessage="To use Fleet, you must enable the following features:
-          {space}- Enable Elasticsearch API keys.
-          {space}- Enable TLS to secure the communication between Agents and Kibana.
-          {space}- Set the encryption key for encrypted saved objects.
-          "
-            values={{
-              space: <EuiSpacer size="m" />,
-            }}
+            id="xpack.ingestManager.setupPage.missingRequirementsElasticsearchTitle"
+            defaultMessage="In your Elasticsearch configuration, enable:"
           />
-        </EuiText>
-        <EuiSpacer size="l" />
-      </>
-    ) : (
-      <>
-        <EuiSpacer size="m" />
-        <EuiIcon type="lock" color="subdued" size="xl" />
-        <EuiSpacer size="m" />
-        <EuiTitle size="l">
-          <h2>
+          <EuiSpacer size="l" />
+          <RequirementItem isMissing={false}>
+            <FormattedMessage
+              id="xpack.ingestManager.setupPage.elasticsearchSecurityFlagText"
+              defaultMessage="Elasticsearch security. Set {securityFlag} to {true} ."
```

Comment:
in designs, Elasticsearch security is linked. maybe it's meant to link to https://www.elastic.co/guide/en/elasticsearch/reference/current/configuring-security.html?